canonical-web-and-design/ubuntu.com
# Move discourse engage pages instance back to app.py

## Done

This is a fix for https://github.com/canonical-web-and-design/ubuntu.com/pull/11006
Currently engage pages are intermittently 404ing https://ubuntu.com/engage/migrating-to-ubuntu-lts-six-facts-for-centos-users
Apparently discourse module instasnce MUST be in the app.py where all the routes are, to avoid being overriden by `template_finder`.

## QA

New [Engage pages topic index](https://discourse.ubuntu.com/t/engage-pages-for-ubuntu-com/18033) and old [takeovers topic index](https://discourse.ubuntu.com/t/takeovers-for-ubuntu-com/17229)

- Check https://ubuntu-com-11034.demos.haus/ make sure takeovers work as usual
- Check https://ubuntu-com-11034.demos.haus/takeovers work as usual
- Check https://ubuntu-com-11034.demos.haus/engage work as usual
- On https://ubuntu-com-11034.demos.haus/engage click on a few engage pages, fill out the form make sure the whole process works as usual. Make sure they NEVER go into 404, as with previous PR, some went to 404 others didn't.

## Issue / Card

Fixes #10299 

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
